website/content/docs/upgrading/upgrade-to-1.16.x.mdx

```diff
@@ -81,6 +81,19 @@ userattr="userprincipalname"
 Refer to the [LDAP auth method (API)](/vault/api-docs/auth/ldap) page for
 more details on the configuration.
 
+### JWT auth login requires bound audiences on the role
+
+JWT auth roles of type "jwt" require the `bound_audiences` claim to match at
+least one of the JWT's `aud` claims. Prior to 1.16.3, the JWT auth method would
+ignore token `aud` claims that were not a list of strings.
```

Comment:
```suggestion
ignore token `aud` claims that were not a list of strings. Starting from 1.16.3, 
it would ignore token `aud` claims that are not a single audience string or a list of strings.
```
I think we should state what changed from 1.16.3. My wording might be confusing here.

Reply:
I think the previous sentence makes that clear. Maybe we can get the education team's input on this? That said, I think your wording should say something like

> Starting from 1.16.3, token `aud` claims can be a single string or a list of strings and they must match at least one of the role's `bound_audiences.

But then maybe the next paragraph should be reworked?

cc @schavis

Reply:
I think saying this could be misleading. 
> Starting from 1.16.3, token `aud` claims can be a single string or a list of strings 

The value type of `aud` in JWT claims follows the specification of JWT, which changed to allow string type independently. I believe users had been sending login requests with single string `aud` prior to 1.16.3. We didn't include that type into our invalidation logic until 1.16.3.

Reply:
Ok, how about I change the wording of the paragraph to this:

> JWT auth roles of type "jwt" require the `bound_audiences` claim to match at
least one of the JWT's `aud` claims. The JWT's `aud` claim can be a single string
or a list of strings as per [RFC 7519 Section 4.1.3](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3).  Prior to 1.16.3, the JWT auth method
would ignore token `aud` claims that were not a list of strings.